shop/src/main/java/ro/msg/learning/shop/domain/entity/Product.java

```diff
@@ -23,10 +25,12 @@ public class Product {
     private BigDecimal price;
     private Double weight;
 
-    @ManyToOne
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
CascadeType.ALL can produce errors in the future chapters. I recommend to do the necessary changes programmatically in the services and remove this annotation.